funfact/initializers.py

```diff
@@ -148,6 +148,7 @@ def stack(initializer, append: bool = True):
     '''Stacks initializers for the purpose of vectorization.
 
     Args:
+        initializer (callable): non-vectorized initializer.
```

Comment:
'non-vectorized' sounds a bit awkward to me. How about 'The initializer to the vectorized'?